e2e/test_helpers_test.go

```diff
@@ -0,0 +1,128 @@
+package e2e
+
+import (
+	"fmt"
+	"github.com/Azure/agentbaker/e2e/config"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestImagesAreFullySpecified(t *testing.T) {
```

Comment:
is this test only going to fun on Windows ?  how is the test seleccted for a specific OS ? should it be named differently ?